content/docs/command-reference/move.md

```diff
@@ -161,3 +165,57 @@ $ tree
     │   └── foo
     └── data3.dvc
 ```
+
+## Example: manually renaming stage outputs
+
+`dvc move` doesn't support renaming of <abbr>outputs</abbr> in a stage, you will
+have to do that manually. Let's look at our sample workspace and `dvc.yaml`:
```

Comment:
I don't think this qualifies as an example, as `dvc move` is not being used. It can be a subsection of the description perhaps (H3) but it should also probably be much shorter. Or it could be a section in `dvc run` (about using its `-f` option for renaming) and link to there. And/or a section in the dvc.yaml spec (https://dvc.org/doc/user-guide/dvc-files-and-directories#dvcyaml-file) for the fully manual method + also link. WDYT?

Reply:
Also writing could improve a little e.g. "renaming of outputs in a stage" is a complicated phrase, not sure... Let's use simple, direct language like "renaming stage outputs". Thanks

> The whole phrase may not be needed though, as if this is moved to the description, it will be repetitive.

Reply:
I think its better suited in this document as it is related to renaming. Moved into Description as it is not a `move` example. The 'dvc run' part can be example under `dvc run` with a link to example here.

Reply:
I'm not so sure... It's not avout `dvc move` at all really. I think it belongs somewhere else. Probably both in dvc.yaml (the manual move) and in `dvc run` (for ita -f option)